src/lang/po/de/Prusa-Firmware-Buddy_de.po

```diff
@@ -471,7 +471,7 @@ msgstr "Kein USB"
 
 #: src/gui/wizard/screen_wizard.c:392
 msgid "Now, let's calibrate\nthe distance       \nbetween the tip    \nof the nozzle and  \nthe print sheet.   "
-msgstr "Jetzt kalibrieren\nwir die Entfernung\nzwischen der Spitze\nder Düse und\ndas Druckblech."
+msgstr "Jetzt kalibrieren\nwir die Entfernung\nzwischen der Spitze\nder Düse und\ndem Druckblech."
```

Comment:
In Prusa Firmware is `Stahlblech` used so it makes sense to keep wording:
`"Jetzt kalibrieren\nwir die Entfernung\nzwischen der Spitze\nder Düse und\ndem Stahlblech."

The change from `...und\ndas...` to `...und\ndem` is correct proposal.
"Der Dativ ist dem Genitiv sein Tod" is a know German sentence for the complicated rules in German.